### PR TITLE
Updated to ember 2.15.1 and...

### PR DESCRIPTION
**Pull request by EstesE**
Updated to Ember 2.15.1 and; added another rental in mirage and added a button to navigate back to the search (rental) page.

### CODE REVIEW

**Comment by acorncom** [comment]
hey @Mithrilhall, thanks for the PR! One of the complex things about making changes to this repo is we also have to make sure we keep the tutorial in the guides in sync. Would you mind sending in a PR to the guides as well for these changes?

File: app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.ml10 {
+    margin-left: 10px;
+}
```
Comment (acorncom) [inline app/styles/app.css]: If we want to make this change, we should aim to make it on the style add-on: https://github.com/toddjordan/ember-cli-tutorial-style

File: mirage/config.js [hunk 2]
```diff
@@ -12,6 +12,19 @@ export default function() {
...
     {
       type: 'rentals',
+      id: 'boston-lofts',
+      attributes: {
+        title: "New England Charm",
+        owner: "Charlie Bucket",
+        city: "Boston",
+        "property-type": "Loft Apartment",
+        bedrooms: 3,
+        image: "https://upload.wikimedia.org/wikipedia/commons/b/bc/Bank_building_denver_night.jpg",
```
Comment (acorncom) [inline mirage/config.js]: We're using a Denver image for Boston? 😛

File: mirage/serializers/application.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import { JSONAPISerializer } from 'ember-cli-mirage';
+
+export default JSONAPISerializer.extend({
+});
```
Comment (acorncom) [inline mirage/serializers/application.js]: Don't think this file is needed, as it's the default for Mirage?

**Comment by EstesE** [comment]
@acorncom Sorry about leaving in the wrong image for Boston. :disappointed:

I've removed Boston from the mirage config, removed the class I created and also removed the link-to I added to go back to the index page. So...it should only be upgraded to Ember 2.15.1. If I find time I'll add Boston back and send in a PR for the guides so they're in sync.

**Comment by jenweber** [comment]
Hey @Mithrilhall, we just upgraded straight to 2.17 in a different PR that was recently merged. Sorry we overlooked that your edits were done to just be the version # back in the day. I'm closing this, but if you want to add another rental in a new PR, that would be welcome. I think having another example would highlight the helper functionality. Get in touch if you have ideas about how to improve the tutorial!

**Pull request closed without merging**